pkg/shared/constants/external.go

```diff
@@ -0,0 +1,164 @@
+package constants
```

Comment:
I get that this is just a move - but why is a list of functions in a package named constants?